Gemfile

```diff
@@ -4,4 +4,5 @@ gem 'github-pages'
 gem 'html-proofer'
 gem 'rack-contrib', '~> 1.1.0'
 gem 'rake'
-gem 'jekyll-octicons'
\ No newline at end of file
+gem 'jekyll-octicons'
+gem 'html-proofer'
```

Comment:
This gem is already up in line 4. I don't think we need it again. 
```suggestion

```

Reply:
@hectorsector Thanks for catching that